### ISSUE TITLE
3.14 Support

### ISSUE THREAD

**Issue by paubanon** [mb1986/rm-hacks#385]

**Comment by arturo90** [comment]
also looking for info this . I have since turned off the auto update feature to get rm hacks

**Comment by Ownezx** [comment]
I'm currently running 3.9.3.1986 and just got an email that the cloud support will drop for that version in 30 days. I would also be interested in a bump of rm-hacks 0.0.10 to 3.14 if/when there is the bandwidth for that to be implemented.

In any case thank you so much for the work put into this, it always has been very appreciated.

**Comment by Eeems** [comment]
> I'm currently running 3.9.3.1986 and just got an email that the cloud support will drop for that version in 30 days. I would also be interested in a bump of rm-hacks 0.0.10 to 3.14 if/when there is the bandwidth for that to be implemented.
> 
> In any case thank you so much for the work put into this, it always has been very appreciated.

3.11.2.5 is currently supported by rm-hacks, and can be reached by just upgrading the OS using the menu, as it's the stopping point before the new upgrade mechanism is used.

**Comment by Ownezx** [comment]
Thank you for your answer, I ended up using RCU to backup and update the firmware and am now probably up and running for a few extra months!

**Comment by mb1986** [comment]
I was stuck on 3.12.4.4 and had some issues at work I had to deal with. I'll try to make you happy again soon!

**Comment by Ownezx** [comment]
Take the time you need, 3.11.2.5 will likely still be supported for a bit on remarkable connect so anyone who really wants to run the hack will be able to.

**Comment by maxanthem** [comment]
> Take the time you need, 3.11.2.5 will likely still be supported for a bit on remarkable connect so anyone who really wants to run the hack will be able to.

Can you confirm that as of today, 3.11.2.5 is still supported with the remarkable cloud? :-)

**Comment by Ownezx** [comment]
Yup. I updated in september from 3.9 because i had an email giving 30 day warning of connect being phased out for my version. I think that it will be the same for 3.11.2.5 and that we'll get an email before it deactivates.
